test/testother.cpp

```diff
@@ -10482,7 +10504,7 @@ class TestOther : public TestFixture {
               "    int x[] = { 10, 10 };\n"
               "    f(x[0]);\n"
               "}");
-        ASSERT_EQUALS("", errout.str());
+        ASSERT_EQUALS("[test.cpp:3]: (style) Variable 'x' can be declared as const array\n", errout.str());
```

Comment:
Const on local variables should use a different ID.

Reply:
The ID is `constVariable`, which seems correct.